cluster/gce/gci/scaleoutserver-helper.sh

```diff
@@ -85,7 +85,7 @@ function create-proxy-instance-internal() {
       --zone "${ZONE}" \
       --machine-type "${MASTER_SIZE}" \
       --image-project="${PROXY_IMAGE_PROJECT:-ubuntu-os-cloud}" \
-      --image "${PROXY_IMAGE:-ubuntu-1804-bionic-v20201014}" \
+      --image "${PROXY_IMAGE:-ubuntu-2004-focal-v20211202}" \
```

Comment:
if PROXY_IMAGE is safely assumed, no need to have the default fallback, which could simplify code a bit.

Reply:
thanks and updated.